src/modules/RegAuth/Registration/containers/RegistrationCreateNewUserContainer.ts

```diff
@@ -4,11 +4,12 @@ import {reduxForm} from 'redux-form';
 import {sendData} from '../actions/RegistrationActions';
 
 import RegistrationCreateNewUser from '../components/RegistrationCreateNewUser';
-import {isFetching} from '../selectors/selectors';
+import {getRegistrationError, isFetching} from '../selectors/selectors';
 
 export default reduxForm({form: 'registration'})(connect(
 	(state: any) => ({
 		isFetching: isFetching(state),
+		registrationError: getRegistrationError(state),
```

Comment:
getRegistrationError

Reply:
I don't get it 😕

Reply:
there should also be getIsFetching selector not just isFetching, because we identify selectors by this "get" key. You can pass props as isFetching and registrationError, but selectors should start with "get"